_docs/releasing_version.md

```diff
@@ -20,13 +32,15 @@ ruby 1.8 support has been dropped.)
 If necessary, run `bundle install` before continuing. If you want you can also only install the needed gems:
 
 ```bash
-bundle install --path .vendor/ --without system_tests development
+# You can add --jobs with the number of cores you have to run this much faster in parallel.
+bundle install --path .vendor --jobs 8
 ```
 
 And in case you installed the gems before:
 
 ```bash
-bundle install --path .vendor/ --without system_tests development; bundle update; bundle clean
+rm -fr .vendor
```

Comment:
If you're purging (and I really think you shouldn't need to), you should remove `Gemfile.lock`. In general I find removing `Gemfile.lock` is more useful than removing the installed gems.

I'd say we should recommend to only do this in case there are problems.

Reply:
If bundling to a directory outside of the project, it should be enough to update the bundle:

```suggestion
bundle update
```

I had a few times to `rm Gemfile.lock` before installing again in the past, but I think it's much a bug of older versions of bundler rather than something we should recommend people to do?  Maybe we can split this in 2:
1. How to update;
2. What to do if something goes wrong.

Reply:
I still have it every now and then, but it's getting less frequent yet. :+1: for splitting it as you suggested.